tests/test_filters.py

```diff
@@ -177,6 +182,7 @@ class SearchListView(generics.ListAPIView):
 
         request = factory.get('/', {'search': r'^\w{3}$', 'title_only': 'true'})
         response = view(request)
+        print(response.data)
```

Comment:
```suggestion
```

Reply:
I guess we could remove it as it seems to be added on debug purpose only?

Reply:
Yeah, sorry.